Torch.Server/TorchServer.cs

```diff
@@ -182,7 +183,8 @@ private static void CheckServerResponding(object state)
             if (!mre.WaitOne(TimeSpan.FromSeconds(Instance.Config.TickTimeout)))
             {
                 var mainThread = MySandboxGame.Static.UpdateThread;
-                mainThread.Suspend();
+                if (mainThread.ThreadState == System.Threading.ThreadState.Running)
```

Comment:
The main thread could possibly be in a second state, `ThreadState.WaitSleepJoin`.  Probably should check for that possibility.